oklayoutinflator/src/main/java/tech/okcredit/layout_inflator/OkLayoutInflater.kt

```diff
@@ -18,7 +21,7 @@ import kotlinx.coroutines.*
  * 3. does not support setting a {@link LayoutInflater.Factory} nor {@link LayoutInflater.Factory2}. Here we are Adding support
  * 4. There is no way to cancel ongoing inflation. here we are exposing coroutine job cancel method
  */
-class OkLayoutInflater(context: Context) {
+class OkLayoutInflater(context: Context): DefaultLifecycleObserver {
```

Comment:
Since this class is now an Observer, don't we also need to pass an instance of this class to `lifecycle.addObserver()` ?

Reply:
i have added the missing files.